src/open_inwoner/search/tests/test_page.py

```diff
@@ -319,6 +333,8 @@ def _test_search(checkbox_name, expected_text):
                 expected_text
             )
 
+        print("Tag 1", self.product1.name)    
```

Comment:
The 'name' should be coming from `self.tag1 = TagFactory.create(name="Tag 1")` I think? 
I'd really ask @pi-sigma if he can help out with this failing test. 
Since all the proper elements seem to be there, so I am unsure how to solve this 😞